spec/main-spec.js

```diff
@@ -0,0 +1,26 @@
+describe("atom-ide-datatip", () => {
+
+    beforeEach(async () => {
+        jasmine.attachToDOM(atom.views.getView(atom.workspace))
+        // Activate activation hook
+        atom.packages.triggerActivationHook('core:loaded-shell-environment');
+        atom.packages.triggerDeferredActivationHooks();
+        // Activate package-deps
+        await atom.packages.activatePackage("busy-signal")
+        await atom.packages.activatePackage("atom-ide-markdown-service")
+    })
+
+    it("Activation", async function () {
+
+        // This makes the log visible again from the command line.
+        spyOn(console, 'log').and.callThrough();
```

Comment:
Is a call through required? I believe log works without a call through

Reply:
Yes, we need it.
https://stackoverflow.com/a/54334807/7910299